SPECS/qemu.spec

```
@@ -227,6 +231,9 @@ cp -r scripts/qmp %{buildroot}%{_datarootdir}/qemu
 %{?_cov_results_package}
 
 %changelog
+* Mon Feb 03 2025 Lucas Ravagnier <lucas.ravagnier@vates.tech> - 4.2.1-5.2.10-1
```

Please update the date and fix the release version:
```suggestion
* Mon Feb 10 2025 Lucas Ravagnier <lucas.ravagnier@vates.tech> - 4.2.1-5.2.10.1
```